src/js/controller/model.js

```diff
@@ -223,12 +228,20 @@ define([
                 this.set('flashBlocked', false);
             }
 
+            // If there is no current provider, then the video tag might have a preset playback rate...
+            if (!_provider) {
+                this.set('defaultPlaybackRate', _currentProvider.getPlaybackRate());
+            }
```

Comment:
Why does not having `_provider` set mean we should set 'defaultPlaybackRate'?

'defaultPlaybackRate' should be controlled by the user. 'playbackRate' should reflect the provider's rate.